app/styles/partials/footer.scss

```diff
@@ -2,6 +2,9 @@
   display: flex;
   min-height: 100vh;
   flex-direction: column;
+  .main-container {
```

Comment:
@sumedh123 Not all screen sizes have this issue. So figure out which all have the issues and fix accordingly. 

And when using media queries to properly support different device width's, ensure you follow Semantic-UI's device widths (https://semantic-ui.com/elements/container.html)

<img width="1680" alt="screen shot 2017-05-07 at 12 29 36 am" src="https://cloud.githubusercontent.com/assets/2404372/25775152/457b30a0-32bc-11e7-883a-05d2837d4f5d.png">